docs/sources/getting-started/get-started-grafana-prometheus.md

```diff
@@ -14,43 +14,98 @@ Prometheus is an open source monitoring system for which Grafana provides out-of
+
+When you run node_exporter locally, navigate to `http://localhost:9100/metrics` to check that it is exporting metrics.
 
 > **Note**: The instructions in the referenced topic are intended for Linux users. You may have to alter the instructions slightly depending on your operating system. For example, if you are on Windows, use the [windows_exporter](https://github.com/prometheus-community/windows_exporter) instead.
 
 #### Install and configure Prometheus
 
-1. Install Prometheus following instructions in the [Installation](https://prometheus.io/docs/prometheus/latest/installation/) topic in the Prometheus documentation.
-
-1. Configure Prometheus to monitor the hosts where you installed node_exporter. In order to do this, modify Prometheus's configuration file. By default, Prometheus looks for the file `prometheus.yml` in the current working directory. This behavior can be changed via the `--config.file` command line flag. For example, some Prometheus installers use it to set the configuration file to `/etc/prometheus/prometheus.yml`. Here is an example of the code you will need to add.
+1. After [downloading Prometheus](https://prometheus.io/download/#prometheus), extract it and navigate to the directory.
 
    ```
-    # A scrape configuration containing exactly one endpoint to scrape from node_exporter running on a host:
-    scrape_configs:
-        # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
-        - job_name: 'node'
+   tar xvfz prometheus-*.tar.gz
+   cd prometheus-*
+   ```
 
-        # metrics_path defaults to '/metrics'
-        # scheme defaults to 'http'.
+1. Locate the `prometheus.yml` file in the directory.
 
-          static_configs:
-          - targets: ['<hostname>:9100']
-   ```
+1. Modify Prometheus's configuration file to monitor the hosts where you installed node_exporter.
+
+By default, Prometheus looks for the file `prometheus.yml` in the current working directory. This behavior can be changed via the `--config.file` command line flag. For example, some Prometheus installers use it to set the configuration file to `/etc/prometheus/prometheus.yml`.
+
+The following example shows you the code you should add. Notice that static configs targets are set to `['localhost:9100']` to target node-explorer when running it locally.
+
+```
+ # A scrape configuration containing exactly one endpoint to scrape from node_exporter running on a host:
+ scrape_configs:
+     # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
+     - job_name: 'node'
+
+     # metrics_path defaults to '/metrics'
+     # scheme defaults to 'http'.
+
+       static_configs:
+       - targets: ['localhost:9100']
+```
 
 1. Start the Prometheus service:
+
    ```
     ./prometheus --config.file=./prometheus.yml
    ```
 
+1. Confirm that Prometheus is running by navigating to `http://localhost:9090`.
```

Comment:
Maybe we want to call out that 9090 is the default port, but could change depending on their prometheus installation? Either that or add the prometheus config above that includes the 9090 target, so it's obvious where that's coming from (not everyone will know that 9090 is the default port for prom like we do)